runtime/base/clean.sh

```diff
@@ -0,0 +1,37 @@
+# ------------------------------------------
+# This script cleans extra files from /opt
+# to keep the layers as small as possible.
+# ------------------------------------------
+
+# Stop on error
+set -e
+# Treat unset variables and parameters as an error.
+set -u
+
+# Strip all the unneeded symbols from shared libraries to reduce size.
+find /opt/bref -type f -name "*.so*" -o -name "*.a"  -exec strip --strip-unneeded {} \;
+find /opt/bref -type f -executable -exec sh -c "file -i '{}' | grep -q 'x-executable; charset=binary'" \; -print|xargs strip --strip-all
+
+# Cleanup all the binaries we don't want.
+find /opt/bref/bin -mindepth 1 -maxdepth 1 ! -name "php" ! -name "php-fpm" -exec rm {} \+
+find /opt/bin -mindepth 1 -maxdepth 1 ! -name "php" ! -name "php-fpm" -exec rm {} \+
```

Comment:
Slight change here: `pecl` is now removed for layers (wasn't working anyway since PEAR was completely removed).